mavros/src/plugins/setpoint_raw.cpp

```diff
@@ -217,6 +238,47 @@ class SetpointRawPlugin : public plugin::PluginBase,
 				body_rate,
 				req->thrust);
 	}
+
+	void rpyt_cb(const mav_msgs::RollPitchYawrateThrustConstPtr msg)
+	{
+	
+		if(ignore_rpyt_messages_){
+			ROS_FATAL("Recieved roll_pitch_yaw_thrust_rate message, but ignore_rpyt_messages_ is true: "
+	 	        "the most likely cause of this is a failure to specify the thrust_scaling_factor,	"
+			"yaw_rate_scaling_factor or system_mass_kg parameters");
+		     	return;
+		}
+
+		//the masks are much more limited than the docs would suggest so we don't use them
+		uint8_t type_mask = 0;
```

Comment:
Why? Why not follow what is available in `setpoint_attitude`? Setting to 0 actually means that every dimension is considered, which is not actually true.

Reply:
This I didn't re-check. Where is masks doc?

Reply:
http://mavlink.org/messages/common/#SET_POSITION_TARGET_LOCAL_NED